test/e2e/vault-decryption-chrome.spec.js

```diff
@@ -121,7 +136,23 @@ describe('Vault Decryptor Page', function () {
         // fill the input field with storage recovered from filesystem
         await driver.clickElement('[name="vault-source"]');
         const inputField = await driver.findElement('#fileinput');
-        inputField.press(await getExtensionStorageFilePath(driver));
+
+        const filePath = await getExtensionStorageFilePath(driver);
+
+        // Log the file size for debugging purposes
+        await logFileSize(filePath);
+        await inputField.press(filePath);
+
+        const fileLoaded = await driver.isElementPresent({
+          tag: 'span',
+          text: '❌ Can not read vault from file',
+        });
+
+        // Retry-logic if file upload fails
+        if (!fileLoaded) {
```

Comment:
Hi, I have a question about the logic. The value of the `fileLoaded` variable will be true if the element "Can not read vault from file" appears. Is this intended? It seems like it should be reversed. But maybe I misunderstood.

Reply:
hey @chloeYue that's an awesome catch :heart_eyes:  I've pushed a fix for that.
I'm now trying to see if there could be an alternative way of fixing this on the root cause - based on [this comment] from David (https://consensys.slack.com/archives/CTQAGKY5V/p1724395549815109?thread_ts=1724321902.861359&cid=CTQAGKY5V) If so, I'll update the PR one more time, if not, then we can leave this as a mitigation. I'll share my updates soon :pray: